rplugin/python3/deoplete/source/lsp.py

```diff
@@ -52,12 +52,7 @@ def __init__(self, vim):
         self.vim.vars['deoplete#source#lsp#_prev_input'] = ''
 
     def gather_candidates(self, context):
-        if not self.vim.call('exists', '*lsp#add_server_config'):
-            return []
-
-        if not self.vim.call('luaeval',
-                             'vim.lsp.client_has_started(_A.filetype)',
-                             {'filetype': context['filetype']}):
+        if not self.vim.call('exists', '*lsp#add_filetype_config'):
```

Comment:
This will be removed. https://github.com/neovim/neovim/pull/11430/commits/2d580756ca707945c01703d404e10f4bf412a72c

Reply:
I have fixed the check.